pkg/cmd/create/work/cmd.go

```diff
@@ -62,6 +62,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	o.ClusterOption.AddFlags(cmd.Flags())
 	cmd.Flags().StringVar(&o.Placement, "placement", "", "Specify an existing placement with format <namespace>/<name>")
 	cmd.Flags().BoolVar(&o.Overwrite, "overwrite", false, "Overwrite the existing work if it exists already")
+	cmd.Flags().BoolVarP(&o.UseReplicaSet, "replicaset", "rs", false, "Create ManifestworkReplicase instead of ManifestWork")
```

Comment:
Better help message like "Create ManifestWorK for the associated placement's clusters using manifestWorkReplicaSet"